src/Assets.jl

```diff
@@ -53,6 +53,14 @@ function Position{Cash{C,N}}(a) where {C,N}
     Position{Cash{C,N},T}(T(a))
 end
 
+macro currency(ccys)
```

Comment:
I'm wondering if this should be `@cash`.

It might also be good to have

```julia
upnam = Symbol(uppercase(string(nam)))
Base.eval(__module__, :( const $upnam = cash($(QuoteNode(upnam))) ) )
```

Reply:
`@cash` would be more consistent since we also have `cash"USD"`. We intentional distinguish "currency" from "cash" when referring to it as a financial instrument.